deploy/charts/cert-manager/values.yaml

```diff
@@ -91,6 +91,16 @@ securityContext: {}
 #   runAsUser: 1000
 #   runAsNonRoot: true
 
+# Container Security Context
```

Comment:
Can you expand this out to be `securityContext to be set on the 'controller' component container` (and similarly across the other containers)

Reply:
Done.